_posts/2023-06-09-flat-object.md

```diff
@@ -0,0 +1,365 @@
+---
+layout: post
+title:  "Use flat object in OpenSearch"
+authors:
+- mingshl
+- kolchfa
+date: 2023-06-09
+categories:
+ - technical-post
+meta_keywords: 
+meta_description: 
```

Comment:
Adding meta: 

**meta_keywords:** flat object field type, Mapping with flat object, OpenSearch mapping
**meta_description:** Explore how the flat object field type simplifies mapping data structures introduced in OpenSearch 2.7 and enhances the search experience.